app/Console/Commands/Game.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Cache\Repository as CacheRepository;
+use Illuminate\Contracts\Config\Repository as ConfigRepository;
+
+class Game extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'game1';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'russian roulette game';
+
+    private CacheRepository $cacheRepository;
+
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(CacheRepository $cacheRepository, ConfigRepository $configRepository)
+    {
+        parent::__construct();
+        $this->cacheRepository = $cacheRepository;
+        $this->configRepository = $configRepository;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+
+
+    public function handle()
+    {
+
+        $lastVisit = $this->cacheRepository->get('lastGameVisit', 'never');
+        $diedStats = $this->cacheRepository->get('diedCount', 0);
+        $surviveStats = $this->cacheRepository->get('surviveCount', 0);
+
+
+        $this->cacheRepository->set('lastGameVisit', date(\DateTime::ATOM),);
+        
+        $this->info("Game visit last time: {$lastVisit}");
+
+        $bad_numbers = $this->cacheRepository->get('Bad-numbers', []);
+
+        $executionCicle = True;
+        $count = 1;
+        $table = [];
+        $table2 = [];
+        $answer = 'survive';
+        $res = 0;
+
+        while ($executionCicle) {
+
+            $random = $random = random_int(1, 6);
+
+            
+            $bad_numbers[$random] = $bad_numbers[$random] ?? 0;
+            $bad_numbers[$random]++;
+    
+            $this->cacheRepository->set('Bad-numbers', $bad_numbers, 86400);
+    
+
+           
+            $res = $this->ask("Pull the triger!");
+
+            if ($random === 1) {
+                $answer = 'die';
+                $table[] = [$answer, $count, $random];
+                $this->table(['answer', 'attempts', 'random_nr'], $table);
```

Comment:
Representation of statistics, was not changed.

Reply:
add bullet in chamber, and show when player was killed

Reply:
still representing result in same command.

Reply:
in table 2 there are another results $this->table(['bad nr.', 'dad times'], $table2);